app/helpers/battles_helper.rb

```diff
@@ -1,11 +1,16 @@
 module BattlesHelper
-  def text_with_images(text)
+  def text_with_images(text, flex=true)
     video_regex = /https?:\/\/\S+\.(?:gifv|webm|mp4)/i
     image_regex = /https?:\/\/\S+\.(?:gif|jpg|jpeg|png)/i
     text.split(/(?<=\s)/).map do |match|
       if match.match(video_regex)
-        substitution = match.sub(/gifv$/, "webm")
-        "<br>#{video_tag(substitution, autoplay:"autoplay", loop:"loop")}<br>"
+        if flex
+          substitution = match.sub(/gifv$/, "webm")
+          "<div class='flex-video'>#{video_tag(substitution, autoplay:"autoplay", loop:"loop")}</div>"
+        else
```

Comment:
Oops, the default is supposed to be false.  Which explains the reason why I called it specifying "true" in the show view.